crates/fluvio-spu/src/services/public/stream_fetch.rs

```diff
@@ -80,6 +95,7 @@ impl StreamFetchHandler {
             debug!("Has WASM payload: {}", msg.wasm_payload.is_some());
 
             let smartstream = if let Some(payload) = msg.wasm_payload {
+                std::thread::sleep(std::time::Duration::from_secs(40));
                 let wasm = &payload.wasm.get_raw()?;
                 let module = sm_engine.create_module_from_binary(wasm).map_err(|err| {
```

Comment:
This operation in the test is very fast so, sometimes we don't notice the error, in cloud it usually takes more than 10 seconds (but sometimes is also fast).